src/lib/stitching/convection/schema.ts

```diff
@@ -0,0 +1,35 @@
+import { createConvectionLink } from "./link"
+import {
+  makeRemoteExecutableSchema,
+  transformSchema,
+  RenameTypes,
+} from "graphql-tools"
+import { readFileSync } from "fs"
+
+export const executableConvectionSchema = async () => {
+  const convectionLink = createConvectionLink()
+  const convectionTypeDefs = readFileSync("src/data/convection.graphql", "utf8")
+
+  // Setup the default Schema
+  const schema = await makeRemoteExecutableSchema({
+    schema: convectionTypeDefs,
+    link: convectionLink,
+  })
+
+  // Remap the names of certain types from Convection to fit in the larger
+  // metaphysics ecosystem.
+  const remap = {
+    Submission: "ConsignmentSubmission",
+    Category: "SubmissionCategoryAggregation",
+    Asset: "SubmissionCategoryAsset",
+    State: "SubmissionStateAggregation",
+  }
+
+  // Return the new modified schema
+  return transformSchema(schema, [
+    new RenameTypes(name => {
+      const newName = remap[name] || name
+      return newName
+    }),
+  ])
```

Comment:
This is just a simple rename for the convection types to be the same as what exists inside MP today.

Reply:
Cool, makes sense.